int8 id1
int8 id2
float64[] path
geometry_msgs/Point[] path_points
